### PR TITLE
Feature/light approximation

### PR DESCRIPTION
**Pull request by benmaidel**
added new light mode: sets the colors of the led ring depending to the laserscan distances.
added led_offset param: defines where the first led starts

### CODE REVIEW

File: cob_light/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_light)
 
-find_package(catkin REQUIRED COMPONENTS actionlib_msgs actionlib diagnostic_msgs message_generation roscpp std_msgs visualization_msgs)
+find_package(catkin REQUIRED COMPONENTS actionlib_msgs actionlib diagnostic_msgs message_generation roscpp std_msgs visualization_msgs sensor_msgs)
```
Comment (fmessmer) [inline cob_light/CMakeLists.txt]: alphabetical order

File: cob_light/common/src/modeFactory.cpp [hunk 2]
```diff
@@ -107,67 +108,73 @@ Mode* ModeFactory::create(cob_light::LightMode requestMode, IColorO* colorO)
 
 		case cob_light::LightMode::SEQ:
```
Comment (fmessmer) [inline cob_light/common/src/modeFactory.cpp]: this file has a tabs vs. spaces issue

File: cob_light/msg/LightMode.msg [hunk 3]
```diff
@@ -10,6 +10,7 @@ uint8 SEQ =            6           # executes one sequence after another as defi
 uint8 CIRCLE_COLORS =  7           # circle through specific colors, if one color is set only one color will circle
                                    # if more than one color is set, that this colors will circle through
 uint8 SWEEP =          8           # circle color from front to back on both sides
+uint8 DIST_APPROX =    9           # circle color from front to back on both sides
```
Comment (fmessmer) [inline cob_light/msg/LightMode.msg]: comment is copy-and-paste....proper description

File: cob_light/package.xml [hunk 4]
```diff
@@ -23,6 +23,7 @@
   <depend>roscpp</depend>
   <depend>std_msgs</depend>
   <depend>visualization_msgs</depend>
+  <depend>sensor_msgs</depend>
```
Comment (fmessmer) [inline cob_light/package.xml]: alphabetical order

**Comment by NadiaHG** [comment]
Visual Review OK

**Comment by fmessmer** [comment]
LGTM now....

**Pull request merged**